### PR TITLE
Add windows resource files

### PR DESCRIPTION
**Pull request by wkbrd**
Pull request for issue #1950

### CODE REVIEW

File: ACE/ACEXML/apps/svcconf/ACEXML_XML_Svc_Conf_Parser.rc [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ FILEFLAGS 0x0L
+ FILEOS 0x4L
+ FILETYPE 0x1L
+ FILESUBTYPE 0x0L
+BEGIN
+    BLOCK "StringFileInfo"
+    BEGIN
+        BLOCK "040904B0"
+        BEGIN
+            VALUE "FileDescription", "ACE\0"
```
Comment (jwillemsen) [inline ACE/ACEXML/apps/svcconf/ACEXML_XML_Svc_Conf_Parser.rc:16]: Shouldn't be FileDescription and InternalName be different for each DLL?

Comment (wkbrd) [reply ACE/ACEXML/apps/svcconf/ACEXML_XML_Svc_Conf_Parser.rc]: Updated. Thank you!

File: ACE/ACEXML/apps/svcconf/ACEXML_XML_Svc_Conf_Parser.rc [hunk 2]
```diff
@@ -0,0 +1,30 @@
+#include "..\..\..\ace\Version.h"
```
Comment (jwillemsen) [inline ACE/ACEXML/apps/svcconf/ACEXML_XML_Svc_Conf_Parser.rc:1]: Use / instead of \ in case someone tries to cross compile from Linux

Comment (wkbrd) [reply ACE/ACEXML/apps/svcconf/ACEXML_XML_Svc_Conf_Parser.rc]: Updates just pushed.

**Comment by wkbrd** [comment]
@jwillemsen : One of the Linux builds was apparently canceled.  Is it required that this be successful?  If so, how do we trigger this CI to happen again?

**Comment by jwillemsen** [comment]
Looks a glitch on github CI, safe to ignore given all other builds

**Pull request merged**